src/platform/unittest/DataPathTest.cpp

```diff
@@ -613,15 +705,18 @@ TEST_P(DataPathTest, UdpData)
     VERIFY_QUIC_SUCCESS(Datapath.GetInitStatus());
     ASSERT_NE(nullptr, Datapath.Datapath);
 
-    auto serverAddress = GetNewLocalAddr();
-    CxPlatSocket Server(Datapath, &serverAddress.SockAddr, nullptr, &RecvContext);
+    auto unspecAddress = GetNewUnspecAddr();
+    CxPlatSocket Server(Datapath, &unspecAddress.SockAddr, nullptr, &RecvContext);
     while (Server.GetInitStatus() == QUIC_STATUS_ADDRESS_IN_USE) {
-        serverAddress.SockAddr.Ipv4.sin_port = GetNextPort();
-        Server.CreateUdp(Datapath, &serverAddress.SockAddr, nullptr, &RecvContext);
+        unspecAddress.SockAddr.Ipv4.sin_port = GetNextPort();
+        Server.CreateUdp(Datapath, &unspecAddress.SockAddr, nullptr, &RecvContext);
     }
     VERIFY_QUIC_SUCCESS(Server.GetInitStatus());
     ASSERT_NE(nullptr, Server.Socket);
-    RecvContext.DestinationAddress = Server.GetLocalAddress();
+
+    auto serverAddress = GetNewLocalAddr();
+    RecvContext.DestinationAddress = serverAddress.SockAddr;
+    RecvContext.DestinationAddress.Ipv4.sin_port = Server.GetLocalAddress().Ipv4.sin_port;
```

Comment:
Might need to use to GetPort/SetPort helpers.

Reply:
This isn't strictly needed since all platforms have a Ipv4.sin_port field, but I'll make the change anyway.

Reply:
Actually, after searching for "sin_port" I see this is a sweeping change and is a problem with the existing code rather than the PR, so let's leave this for its own PR.